tests/virtual_machines_test.go

```diff
@@ -194,8 +216,7 @@ var _ = Describe("Virtual Machines", func() {
 				Expect(err).ToNot(HaveOccurred())
 			})
 		})
-		//2162 test postponed due to issue: https://github.com/k8snetworkplumbingwg/kubemacpool/issues/104
-		PContext("When trying to create a VM after all MAC addresses in range have been occupied", func() {
+		Context("When trying to create a VM after all MAC addresses in range have been occupied", func() {
```

Comment:
we need to be consistent on putting newline between Context or not. I think we should.

Reply:
Most of the Context block has no newline between, I think we should follow.